components/Projects/ProjectSelection/ProjectSelection.js

```diff
@@ -3,20 +3,22 @@ import Link from 'next/link';
 
 const ProjectSelection = () => {
   return (
-    <div className={styles.projSelect}>
```

Comment:
I'm not sure if I understand why some components have to be put in fragments. What motivates this?

Reply:
I added fragments when I included the Head component in individual components.
when i moved head to the main page of each route, i missed to remove the fragments from some components.
Just removinig them now.

Reply:
I noticed some unnecessary fragments in the following files, which were not modified by me earlier
![image](https://user-images.githubusercontent.com/94868003/205488068-b5b9e24d-6dfc-4c99-8a78-64b2b6d847de.png)
![image](https://user-images.githubusercontent.com/94868003/205488142-9d50fa93-588d-48db-b247-9276d9249e96.png)

OurReach.js
Footer.js
OurSponsors.js
HomeNotifications.js
Navbar.js

should I remove those as well? or only the ones I modified.

Reply:
The codebase does need cleanup, but let's do that seperately. We shouldn't be mixing concerns at the moment.